requirements/common.txt

```diff
@@ -140,3 +137,6 @@ zope.interface==4.1.2
 
 # Django extension for static asset pipeline
 django-pipeline==1.6.8
+
+# Needed for iOS push notifications
+git+https://github.com/sharmaeklavya2/PyAPNs.git@remove-simplejson#egg=apns-2.0.1
```

Comment:
Cool!  I guess they haven't released a version with this fix yet, though, but the fact that your change was in an "August Maintenance Release" milestone is a good sign for that.